core/persistence-jpa/src/main/java/org/apache/syncope/core/persistence/jpa/dao/JPAPlainSchemaDAO.java

```diff
@@ -101,6 +107,16 @@ public PlainSchema find(final String key) {
         return query.getResultList();
     }
 
+    @Override
+    public <T extends PlainAttr<?>> boolean hasAttrs(PlainSchema schema, Class<T> reference) {
+        String plainAttrTable = getPlainAttrTable(reference);
+        Query query = entityManager().createNativeQuery("select count(" + plainAttrTable + ".id) from plainschema join "
```

Comment:
Please:
1. convert SQL statements to be uppercase
1. replace `plainSchema` with `JPAPlainSchema.TABLE`
1. ensure to stay within 120 characters, or build will fail